packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -109,12 +110,19 @@ export async function graphFetch(
             siteUrl,
         );
     const url = graphUrl.startsWith("http") ? graphUrl : `https://graph.microsoft.com/v1.0/${graphUrl}`;
+    const retryFilter: RetryFilter<Response> = allowlist([offlineFetchFailureStatusCode, fetchFailureStatusCode]);
+    const finalRetryPolicy: RetryPolicy<Response> = {
+        backoffFn: constantBackoff(1),
```

Comment:
Isn't 1ms a bit too aggressive? Should we increase it to say 100?

Reply:
Yes, wanted it to be 1sec. But it is in ms. So, I think I will make it 1000. :)